features/pom.xml

```diff
@@ -45,102 +50,166 @@
                     </execution>
                 </executions>
             </plugin>
-
         </plugins>
     </build>
-    <dependencies>
-        <dependency>
-            <groupId>org.apache.karaf.features</groupId>
-            <artifactId>framework</artifactId>
-            <type>kar</type>
-            <scope>runtime</scope>
-            <version>${karaf-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.karaf.features</groupId>
-            <artifactId>standard</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${karaf-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.karaf.features</groupId>
-            <artifactId>enterprise</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${karaf-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>hu.blackbelt.karaf.features</groupId>
-            <artifactId>antlr-features</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${antlr-karaf-features-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>hu.blackbelt.karaf.features</groupId>
-            <artifactId>apache-commons-features</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${apache-commons-karaf-features-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>hu.blackbelt.karaf.features</groupId>
-            <artifactId>google-features</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${google-karaf-features-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>hu.blackbelt.karaf.features</groupId>
-            <artifactId>eclipse-emf-features</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${eclipse-emf-karaf-features-version}</version>
-        </dependency>
-    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>verify-feature</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.karaf.tooling</groupId>
+                        <artifactId>karaf-maven-plugin</artifactId>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <enableGeneration>true</enableGeneration>
```

Comment:
Wrong features are verified. These features belongs to apache-commons. The xtext features have to be validated are defines in features.xml.